ccdproc/ccddata.py

```diff
@@ -97,6 +97,11 @@ class CCDData(NDDataArray):
 
     """
     def __init__(self, *args, **kwd):
+        if 'meta' not in kwd:
+            kwd['meta']=kwd.pop('header', None)
```

Comment:
Nitpicky, I know, but should put spaces around equal sign -- see https://www.python.org/dev/peps/pep-0008/